caffe2/contrib/nccl/nccl_ops_test.py

```diff
@@ -62,6 +62,7 @@ def allreduce(*args):
         for output in outputs:
             np.testing.assert_array_equal(outputs[0], output)
             self.assertEqual(outputs[0].tobytes(), output.tobytes())
+        time.sleep(2)
```

Comment:
Please add a comment like this: 

TODO(T60310021): This grace period is temporarily added to allow AMD RCCL to clean up its resources before launching another test. Otherwise it may hang the following tests. Looking at longer term solution.